.github/workflows/build-test.yml

```diff
@@ -147,13 +147,7 @@ jobs:
         done
 
     - name: Install dyff with asdf
-      uses: asdf-vm/actions/install@05e0d2ed97b598bfce82fd30daf324ae0c4570e6  # v3
-      with:
-        # Until v4 is released or we get spam about bash vs go implementations
-        # https://github.com/asdf-vm/actions/issues/587
-        asdf_branch: v0.15.0
-        tool_versions: |
```

Comment:
We still need this line & the one below, just not the 3 above

Reply:
Ah, I interpreted "those 3 lines" as the actual code lines.